channel-email/src/main/java/com/synopsys/integration/alert/channel/email/action/EmailGlobalCrudActions.java

```diff
@@ -46,6 +46,7 @@ public EmailGlobalCrudActions(AuthorizationManager authorizationManager, EmailGl
     public ActionResponse<EmailGlobalConfigModel> create(EmailGlobalConfigModel resource) {
         return configurationHelper.create(
             () -> validator.validate(resource),
+            () -> configurationAccessor.getConfigurationByName(resource.getName()).isPresent(),
```

Comment:
This warrants a new method in the accessor to determine if the config exists (using SQL) as opposed to retrieving the entire config and checking if it's present.

Reply:
yes we should add to the ConfigurationAccessor interface existsConfigurationByName(String name) and then add to each configuration repository existsByName.  Should be a more efficient call.